### DISCUSSION TITLE
Next step for DevX is to be a Catalyst Challenge Team

### DISCUSSION THREAD

**Discussion by aleeusgr** [input-output-hk/Developer-Experience-working-group#69] [Ideas]
==Under review, thanks to Romain Soulat for inputs.

I propose to imagine how is the best way the group to operate in a way that can be captured in the group charter.
For this we need to re-define the role of co-chair.
Co-chairs shared responsibility would be to  review of the [State of the Cardano Developer Ecosystem](https://cardano-foundation.github.io/state-of-the-developer-ecosystem/2022/#).

Ideally it should happen as soon as new survey results are published.

The result of the review should be a Catalyst Challenge proposal: developer experience improvements on Cardano.

**Comment by waalge** [comment]
I think reflecting on the survey is great idea. I dont think it needs to be added to the charter.
Given the resource constraints we generally all suffer from I would suggest that we schedule this reflection for ... when is the next release? and then see what we have the appetite for after that

**Reply by aleeusgr** [reply]
Hi! Many thanks for the input. 
After discussing the idea I changed the header and the title, FYI.
